### DISCUSSION TITLE
RC release versioning

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#59] [General]
As suggested by @hdamker I propose to abort the Apha release to go directly for the RC.

The version for the RC should be: "x.y.z-rc.n" 

I propose this version number for the RC: "version: 0.2.0-rc.1"

For the URL he Guidelines prescribe: v0.yrcn so we have: "{apiRoot}/call-forwarding-signal/v0.2rc1.""

For the release tag: "r1.1"

For the branch: "release-0.2.0-rc.1"

**Comment by hdamker** [comment]
> For the branch: "release-0.2.0-rc.1"

There is no requirement or need to create a release branch anymore (and if, it would be "release-1.1", not based on the API Version). 
The link to the release goes directly to the release tag, e.g. `https://github.com/camaraproject/CallForwardingSignal/blob/r1.1/code/API_definitions/Call_Forwarding_Signal.yaml` or
`https://github.com/camaraproject/CallForwardingSignal/tree/r1.1/`
(but you don't need to be concerned about "tree" vs "blob", GitHub is rewriting it if needed).

**Comment by FabrizioMoggio** [comment]
implemented with: 

- https://github.com/camaraproject/CallForwardingSignal/pull/60
- https://github.com/camaraproject/CallForwardingSignal/pull/62

I will not crate any branch.

**Comment by FabrizioMoggio** [comment]
Because of the Meta-Release review we need to crate a new release and a new Tag:

API Version 0.2.0-rc.2 and r1.2

According to: https://github.com/camaraproject/CallForwardingSignal/pull/68#issuecomment-2244931293

All the relevant files had to be modified:

- [x] - changelog.md: reporting the changes
- [x] - readme.md: updated for the new version and tag
- [x] - API Readiness Checklist: updated for the new version
- [x] - a new scope must be crated. The new release will refer to this new scope.

**Comment by FabrizioMoggio** [comment]
closing because of r1.3
